flake8_quotes/__init__.py

```diff
@@ -54,10 +54,12 @@ def _register_opt(parser, *args, **kwargs):
     @classmethod
     def add_options(cls, parser):
         cls._register_opt(parser, '--quotes', action='store',
-                          parse_from_config=True,
+                          parse_from_config=True, type='choice',
+                          choices=list(cls.INLINE_QUOTES),
```

Comment:
That's what I'm saying. A novice programmer who would like to contribute to this repo might not understand the behavior whereas calling `.keys()` is much more clear